README.md

```diff
@@ -169,12 +166,14 @@ fn code() -> Result<(), Box<Error>> {
     // The method create_document will return, upon success,
     // the document attributes.
     let document_attributes = core.run(
-        client.create_document(&database, &collection, &doc).unwrap()
+        client
+            .create_document(&DATABASE, &COLLECTION, &doc)
+            .execute(),
```

Comment:
`create_document` returns a `CreateDocumentRequest` which needs to be `execute`d to become a `Future`